<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1758</width>
    <height>1468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>결함 데이터 분석기</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Users/mind2k/Downloads/pytrader.png</normaloff>../../Users/mind2k/Downloads/pytrader.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1468</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,10,1,10,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>결함 데이터 학습</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>학습 데이터 설정</string>
            </property>
            <property name="margin">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLoadTrainFile">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>파일 불러오기</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTrain">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>학습하기</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveModel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>모델저장</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableWidget" name="twTrainData">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>작업장코드</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>작업장명</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>작업지시번호</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>품목코드</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>품목명</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>작업장</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>검사측정값</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>합불판정</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>수집일시</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>loss</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>accuracy</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
          <item>
           <widget class="QWidget" name="wdgtLoss" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="wdgtAcc" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>epoch</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>epoch</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,10,1,10,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>결함률 분석</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>분석 데이터 설정</string>
            </property>
            <property name="margin">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLoadTestFile">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>파일 불러오기</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTest">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>분석하기</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="twTestData">
            <column>
             <property name="text">
              <string>작업장코드</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>작업장명</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>작업지시번호</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>품목코드</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>품목명</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>작업장</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>검사측정값</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>결함률</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QWidget" name="wdgtTest" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>index</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
